### ISSUE TITLE
Package of version 1.7.0 is signed with the wrong private key

### ISSUE THREAD

**Issue by koenrh** [bkuhlmann/git-cop#36]
Besides the issue reported in https://github.com/bkuhlmann/git-cop/issues/35, there's another issue that prevents users from installing this gem.

It seems that the package of version 1.7.2 is signed with a private key that does not correspond to the public key in your certificate available at https://www.alchemists.io/gem-public.pem.

The SHA256 hashes of the public key in the  aforementioned certificate, and the one provided in the package are different.

This is the (self-signed) certificate that is provided in the signed gem/package:

```
-----BEGIN CERTIFICATE-----
MIIDNjCCAh6gAwIBAgIBAjANBgkqhkiG9w0BAQUFADBBMQ8wDQYDVQQDDAZicm9v
a2UxGjAYBgoJkiaJk/IsZAEZFgphbGNoZW1pc3RzMRIwEAYKCZImiZPyLGQBGRYC
aW8wHhcNMTcxMDI1MTIxMDU4WhcNMTgxMDI1MTIxMDU4WjBBMQ8wDQYDVQQDDAZi
cm9va2UxGjAYBgoJkiaJk/IsZAEZFgphbGNoZW1pc3RzMRIwEAYKCZImiZPyLGQB
GRYCaW8wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCgryPL4/IbWDcL
fnqpnoJALqj+ega7hSsvvD8sac57HPNLeKcOmSafFiQLAnTmmE132ZlFc8kyZRVn
zmqSESowO5jd+ggFuy1ySqQJXhwgik04KedKRUjpIDZePrjw+M5UJT1qzKCKL2xI
nx5cOKP1fSWJ1RRu8JhaDeSloGtYMdw2c28wnKPNIsWDood4xhbLcY9IqeISft2e
oTAHTHandHbvt24X3/n67ceNjLBbsVZPXCC1C8C8ccjHjA4Tm2uiFoDwThMcPggg
90H6fh0vLFcNAobdPEchbge8tWtfmMPz2+C4yklANn81GA+ANsBS1uwx6mxJoMQU
BNVp0aLvAgMBAAGjOTA3MAkGA1UdEwQCMAAwCwYDVR0PBAQDAgSwMB0GA1UdDgQW
BBRS85Rn1BaqeIONByw4t46DMDMzHDANBgkqhkiG9w0BAQUFAAOCAQEAJn1nVdx9
4FneO7dkqNHOuuSfhXZz05NSfZ6XLmKVeDaoUEO4k3ii8Ho9QJHDCubDZTpjB00l
lrw6XT2WCP+OKI+FLFLAe7RIqoJgZC0VpViPJS1PX28Xuv2Yvp0Aj//rTRB9vuYI
lHadnxmbYKlZCgSqFQbmKfUo1spvYX9SEGaeUkpfhHa59sq0UbIFCfLtQ3hNGPEb
Qkkb8H5hgCTGheH5NJbEfnpCNsAZM4H2lg1HoSmc03xN8ajeTB0Y8VA84U89tKP8
5CXiAU6zuRjW5Q562D7SMKZl8RVVhWxoxV4V3w/3LY922oGdOyYFEjUpp/GE3jEg
l1XwX/SFtXe76Q==
-----END CERTIFICATE-----
```

Find the hash of the public key in that certificate:

```
$ openssl x509 -in gem_cert.pem  -pubkey -noout | openssl rsa -pubin -outform der | openssl dgst -sha256

dab54a57050aa3b53aa9722192ae081b7c22bcb4980891f3f9ae6d7bfae3cded
```

Then find the hash of the public key that is in the certificate that is provided through the aforementioned URL:

```
$ curl https://www.alchemists.io/gem-public.pem | openssl x509  -pubkey -noout | openssl rsa -pubin -outform der | openssl dgst -sha256

bc613ecbbe7a9d0589a9ff91a4638ff39c26029f4daaba7a8440231037a4fd5c
```

Note that the public key hashes are different.

**Comment by bkuhlmann** [comment]
:information_source: Updated issue label to refer to 1.7.0 since 1.7.2 doesn't exist.

**Comment by bkuhlmann** [comment]
Sorry about this and thanks for catching this. I figured out the source of the problem. I had updated the certificates, in a previous issue you reported, but didn't replicate them across the all of the machines I use. I've resolved this now so I can build on all machines without using bad certificates.

This has been fixed in [Version 1.7.1](https://github.com/bkuhlmann/git-cop/releases/tag/v1.7.1) which is available for download now.
